controllers/falcon/falconnodesensor_controller.go

```diff
@@ -765,6 +770,11 @@ func (r *FalconNodeSensorReconciler) conditionsUpdate(condType string, status me
 	})
 
 	if err := r.Status().Update(ctx, nodesensor); err != nil {
```

Comment:
How come we are seeing these? In other words, who else is updating falconnodesensor resource behind our back?

Reply:
The error appears when an object which is sent for reconciliation while the previous updating is still in progress and can happen when the apiserver is getting too many requests so it is ratelimiting them. See https://github.com/kubernetes/kubernetes/issues/28149

I almost would feel like we should not ignore the error in the code and just document that if it happens/exists to safely ignore the error.

Reply:
I have spent some time looking further into this and I came to a conclusion that we indeed are causing this error (almost consistently) by a slight mis-use of the API. What I found is that we may update the nodesensor resource twice without fetching it.

Also, we actively set the condition at least 11 times per roll-out. I think if just didn't pushed the update on a condition, that already has a desired state, we may very well never see the error again.